src/i18n.js

```diff
@@ -21,6 +22,7 @@ export default () => {
     it,
     ja,
     nl,
+    sr,
```

Comment:
Could you please move this below `pt-BR` so that the linting in the CI build will complete? Thanks!

Reply:
Put it on the end is it OK?

Reply:
I think it needs to be between `pt-BR` and `zh-CN` see https://travis-ci.org/github/ProjectMirador/mirador/builds/694337134#L362

Reply:
Great!!!! Do it now! TNX!!!